src/modules/Rooms/components/Video.js

```diff
@@ -34,8 +35,7 @@ const Video = (props) => {
         </div> : null
       }
       <div className='embed-responsive-item'>
-        {/* {!props.callInProgress ? <i className='fa fa-5x fa-user' /> : null} */}
-        {props.callInProgress ? <video ref={attachRemoteStream} autoPlay muted /> : null}
```

Comment:
@JaCorbett, @ericgold, who would have thought that the property "muted" on an html5 element will mute audio...  dur.